staff_features/accessions/accession_spawn_archival_object.feature

```diff
@@ -0,0 +1,26 @@
+Feature: Accession spawn Archival Object
+  Background:
+    Given an administrator user is logged in
+      And an Accession has been created
+      And a Resource has been created
+      And the Accession is opened in edit mode
+  Scenario:  Archival Object form is prefilled with Accession data
+     When the user clicks on 'Spawn'
+      And the user clicks on 'Archival Object' in the spawn dropdown menu
+      And the user selects Resource in the modal
+      And the user clicks on an Archival Object in the Component Position modal
+      And the user clicks on 'Insert spawned component before'
+      And the user clicks on 'Select Component Position'
+     Then the New Archival Object page is displayed
+      And the Archival Object has been spawned from Accession info message is displayed
+      And the Archival Object title is filled in with the Accession Title
+      And the Archival Object publish is set from the Accession publish
+      And the Archival Object notes are set from the Accession Content Description and Condition Description
+      And the following Archival Object forms have the same values as the Accession
```

Comment:
"fields" here would sound better than "forms"

Reply:
I believe that we should use a term to differentiate between fields and forms,
since the values that follow in the data table refer to sub-records and linked records form sections and not specific fields.

If we use the term fields, maybe we should change the data table to be something like:

```
| form_section       | form_field                    |
| Basic Information  | Level of Description          |
| Languages          | Language                      |
| Languages          | Script                        |
| Dates              | Type                          |
| Dates              | Begin                         |
| Extents            | Number                        |
| Extents            | Type                          |
| Finding Aid Data   | Language of Description       |
| Finding Aid Data   | Script of Description         |
```

Or we can re-phrase this step like:
`And the following Archival Object forms have the same field values as the Accession`
`And the following Archival Object form sections have the same field values as the Accession`
`And the following Archival Object sub-records and linked records have the same field values as the Accession`

However, note that the same phrasing has already been merged [here](https://github.com/archivesspace/e2e-tests/pull/18/files#diff-8a36ec8998c185cc564ef1db61b2892aa8f696cb6d2b3c66637bdc03475c2c98R12-R24), so it would be good to keep it consistent, and maybe change it in the future in a new PR.

What are your thoughts on this?